src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService.cs

```diff
@@ -51,6 +51,11 @@ protected override bool IsInConstructorInitializer(ExpressionSyntax expression)
             return expression.GetAncestorOrThis<ConstructorInitializerSyntax>() != null;
         }
 
+        protected override bool IsInExpressionBodiedMember(ExpressionSyntax expression)
+        {
+            return expression.GetAncestorOrThis<ArrowExpressionClauseSyntax>() != null;
```

Comment:
```GetAncestorOrThis<T>``` is an old extension method. Maybe use the real method: ```FirstAncestorOrSelf<T>``` rather than allow this extension method to proliferate?